<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>640</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextUsername">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserMessage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>440</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessage">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>130</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveSettings">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveSettings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>620</y>
      <width>110</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_CreateRoom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="m_chatRoomList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>160</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="m_chatRoomArea">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>440</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_connectToChatRoomBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Chat Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_deleteRoom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>Username</zorder>
   <zorder>TextUsername</zorder>
   <zorder>UserMessage</zorder>
   <zorder>SendMessage</zorder>
   <zorder>SaveSettings</zorder>
   <zorder>Exit</zorder>
   <zorder>m_chatRoomList</zorder>
   <zorder>m_chatRoomArea</zorder>
   <zorder>m_connectToChatRoomBtn</zorder>
   <zorder>m_CreateRoom</zorder>
   <zorder>m_deleteRoom</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
